dockstore-common/src/main/java/io/dockstore/common/yaml/YamlWorkflow.java

```diff
@@ -69,7 +69,7 @@ public void setName(final String name) {
     }
 
     @NotNull
-    @ValidDescriptorLanguage(entryType = EntryType.WORKFLOW, message = "must be a supported descriptor language (\"CWL\", \"WDL\", \"GALAXY\", or \"NFL\")")
+    @ValidDescriptorLanguage(entryType = EntryType.WORKFLOW, message = "must be a supported descriptor language (\"CWL\", \"WDL\", \"GALAXY\", \"SMK\", or \"NFL\")")
```

Comment:
There may already be a ticket for this, but we should adjust our .dockstore.yml templates as well https://docs.dockstore.org/en/stable/assets/templates/workflows/workflows.html#simple-generic-template-for-a-workflow and on the github app landing page https://qa.dockstore.org/github-landing-page?entryType=WORKFLOW